src/swsssdk/configdb.py

```diff
@@ -131,7 +131,7 @@ def __raw_to_typed(self, raw_data):
                     typed_data[key] = raw_data[raw_key]
         return typed_data
 
-    def __typed_to_raw(self, typed_data):
+    def __typed_to_raw(self, typed_data, dkeys=None):
```

Comment:
If dkeys is the output, would it make more sense to make it as return value instead?  I suggest not using dkeys but dfields.

Reply:
In that case, I need to return a tuple {raw, dfields}, and then I need to change all the function which calls __typed_to_raw. I used current approach where caller can pass dfields arg, and then only it is getting populated.

Reply:
@zhenggen-xu : Since in MSFT meeting we decided that we need to allow the deletion of fields, I think it is better to put the same fix in set_entry. Then we can return {raw, dfields} from __typed_to_raw.